tests/test_memory.py

```diff
@@ -97,28 +97,6 @@ def render_foo(self, data):
         assert [r['foo'] for r in table.rows] == [-1, -2]
 
 
-def test_caches():
```

Comment:
I don't have any idea if we rely on this. I doubt it. We never call update on the table. And this test would have been intermittently failing always. Between travis and tox, it fails more often now.